src/tags/control/ClassificationBase.js

```diff
@@ -40,6 +41,23 @@ const ClassificationBase = types.model('ClassificationBase', {
     };
   }).actions(self => {
     return {
+      validate() {
+        if (self.perregion) {
+          return self._validatePerRegion();
+        }
+        else if (self.peritem && isFF(FF_LSDV_4583)) {
+          return self._validatePerItem();
+        }
+        else {
+          return self._validatePerObject();
+        }
+      },
+      validateValue() {
```

Comment:
comment on underscores: feels especially wrong to see those methods one by one with different styles

Reply:
It's great 'cause `validateValue` should be overridden and `_validatePerObject` should not. It's good if you can feel that something here must to be taken in account.

Reply:
add a comment that this method is the one to override in inheritors, not `validate()`